packages/syft/src/syft/server/server.py

```diff
@@ -1585,6 +1585,7 @@ def create_initial_settings(self, admin_email: str) -> ServerSettings | None:
                     show_warnings=self.enable_warnings,
                     association_request_auto_approval=self.association_request_auto_approval,
                     default_worker_pool=get_default_worker_pool_name(),
+                    notifications_enabled=True,
```

Comment:
Quick Question: We set `notifications_enabled` True here but we're not setting the `set_notifier_active_to_true` ? We're setting this the Settings update service https://github.com/OpenMined/PySyft/pull/9049/files#diff-38e8350faa70ff82eb9473d049ab0f1142ca23e9fad17094f6361ea6d7a852daR121 . Which means, that user will have to call the settings.update service again to enable notifier.